### PR TITLE
Additional error conditions. Fixes DOC-1063

### PR DESCRIPTION
**Pull request by lamagnifica**
@shnayder please review (you suggested these additions a few weeks back).
@catong , @mhoeber , @srpearce , I’d like to publish this as soon as
Victor gives the ok.

### CODE REVIEW

File: docs/en_us/dashboard/source/Reference.rst [hunk 1]
```diff
@@ -353,10 +353,22 @@ Error Conditions
...
-Insights is available for every course run.
+new features to the platform. As a result, more data is available for courses
+that are running now, or that ran recently, than for courses that ran in the
+past. Not all data for every value reported by edX Insights is available for
+every course run.
 
-EdX changed the method used to track student enrollments on 3 December 2013. As
-a result, enrollment activity data is not computed for courses created in
-Studio prior to 4 December 2013. Other data is available for those courses.
+The following conditions can result in errors.
```
Comment (mhoeber) [inline docs/en_us/dashboard/source/Reference.rst]: found this lead in a little confusing.  not really conditions.  Perhaps:  "In the following situations, enrollment data may not be available:"

**Comment by mhoeber** [comment]
one comment, otherwise :+1:

**Comment by lamagnifica** [comment]
@srpearce either DD Mon YYYY or DD Month YYYY is the date format specified at Oracle to assure comprehension by the widest possible global audience. It makes sense to me, so I have internalized it. Please add a date format to the style guide!

File: docs/en_us/dashboard/source/Reference.rst [hunk 2]
```diff
@@ -353,10 +353,22 @@ Error Conditions
...
+past. Not all data for every value reported by edX Insights is available for
+every course run.
 
-EdX changed the method used to track student enrollments on 3 December 2013. As
-a result, enrollment activity data is not computed for courses created in
-Studio prior to 4 December 2013. Other data is available for those courses.
+In the following situations, data may not be available in edX Insights. 
+
+* EdX changed the method used to track student enrollments on 3 December 2013.
+  As a result, enrollment activity data is not computed for courses created in
```
Comment (shnayder) [inline docs/en_us/dashboard/source/Reference.rst]: The Enrollment activity graph is visible for old courses now, but shows up as a spike on 3 Dec 2013.

**Comment by lamagnifica** [comment]
Hi @shnayder , please take another look. (This PR predates the PR that removed warnings about enrollment data...)

**Comment by shnayder** [comment]
:ship:

**Pull request merged**